samples/client/petstore/kotlin-jvm-volley/src/main/java/org/openapitools/client/infrastructure/DateAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package org.openapitools.client.infrastructure
+
+import com.google.gson.TypeAdapter
+import com.google.gson.stream.JsonReader
+import com.google.gson.stream.JsonWriter
+import com.google.gson.stream.JsonToken.NULL
+import java.io.IOException
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+class DateAdapter(val formatter: DateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ", Locale.getDefault())) : TypeAdapter<Date>() {
```

Comment:
@alisters this adapter is not used, could you please remove it?

Reply:
@4brunu - This was there already, and i think it might be used by other libraries that use gson like retrofit ? And also other serialisation libraries too (kotlinx). Looking in KotlinClientCodegen.java 

           case gson:
                supportingFiles.add(new SupportingFile("jvm-common/infrastructure/DateAdapter.kt.mustache", infrastructureFolder, "DateAdapter.kt"));

I could comment out or remove this supporting file for the gson serializer if you know it's not used for the other clients ? Otherwise i don't want to break other peoples clients, and i haven't codegen'd this generator for all different library/clients.

Reply:
@alisters thanks for point that out, that is a bug, it's just old code that hasn't been removed.
This PR cleans up that.
https://github.com/OpenAPITools/openapi-generator/pull/11082

Could you please also remove DateAdapter and references to java.util.Date?
Thanks

Reply:
@4brunu - our file is in the samples, and generated code. Will your PR remove this for us ? Other than generating it, and including it as a supporting file i don't think we reference it. Sorry if i'm missing the point here

Reply:
No, my PR only deletes that in the existing projects. Could you please remove this class and also the references to this?

Reply:
Done.

Reply:
@4brunu - i don't think i thought about this. This file is generated, and it's template file is common, in jvm-common/infrastructure/.  So i've removed the DateAdapter.mustache, but it now causes other libraries generated sample code to not compile. For the moment we have regenerated and pushed only _our_ samples - so that all samples still build. Assuming that you're PR will go in before ours, i can merge from master at that point and then all the libraries, ours included will no longer use DateAdapter. Does that sound ok ?

Reply:
Yes, this is indeed a problem.
We have two options, merge first my PR, or I can merge my PR to your PR.

Reply:
I just noticed now, that my PR was merged during the night, so this will no longer be an issue.
But maybe you will need to merge the OpenAPI master branch into your branch.

Reply:
Merged and repushed